xgi/classes/hypergraph.py

```diff
@@ -506,7 +506,7 @@ def add_edges_from(self, ebunch_to_add, **attr):
             i.e. you cannot mix different formats.  The iterables containing edge
             members cannot be strings.
 
-        attr : \*\*kwargs, optional
+        attr : **kwargs, optional
```

Comment:
I made this change because without the backward slashes, Sphinx issues a warning when building the docs.

Reply:
Can you run `make html` from `docs/` to make sure it's not just me?

Reply:
yep, you're right!